frontend/elements/src/module/_groups/cards/play/card/card.ts

```diff
@@ -36,6 +36,10 @@ export class _ extends LitElement {
                     z-index: 1001;
                 }
 
+                :host([side="left"][size="flashcards"]) .front.content {
```

Comment:
@RickyLB this works, but not as expected. The cards in flashcards are reversed for some reason, so the left card has `side="right"` and the right card has `side="left"`. I should have noted this in the task. So this rule actually styles the wrong card correctly.

Instead of having this rule here, consider targeting just `"flashcards"` in styles.ts, and updating the new `--border-radius` css var that you added in that selector. That would make it so that all cards have a border radius of 16px, and just flashcards with the 56px radius.